src/lib.js

```diff
@@ -79,29 +80,15 @@ function downloadVersion(version, os) {
     });
 }
 
-function use(version) {
-    var srcPath = path.join(paths.PSVM_VERSIONS, version, 'purescript'),
-        destPath = path.join(paths.PSVM_CURRENT_BIN),
-        promises = [
-            util.copy(path.join(srcPath, 'psc'), path.join(destPath, 'psc')),
-            util.copy(path.join(srcPath, 'psc-bundle'), path.join(destPath, 'psc-bundle')),
-            util.copy(path.join(srcPath, 'psc-docs'), path.join(destPath, 'psc-docs')),
-            util.copy(path.join(srcPath, 'psc-ide-client'), path.join(destPath, 'psc-ide-client')),
-            util.copy(path.join(srcPath, 'psc-ide-server'), path.join(destPath, 'psc-ide-server')),
-            util.copy(path.join(srcPath, 'psc-publish'), path.join(destPath, 'psc-publish')),
-            util.copy(path.join(srcPath, 'psci'), path.join(destPath, 'psci'))
-        ];
-
-    Promise.all(promises)
-        .then(function () {
-            fs.chmodSync(path.join(destPath, 'psc'), '0777');
-            fs.chmodSync(path.join(destPath, 'psc-bundle'), '0777');
-            fs.chmodSync(path.join(destPath, 'psc-docs'), '0777');
-            fs.chmodSync(path.join(destPath, 'psc-ide-client'), '0777');
-            fs.chmodSync(path.join(destPath, 'psc-ide-server'), '0777');
-            fs.chmodSync(path.join(destPath, 'psc-publish'), '0777');
-            fs.chmodSync(path.join(destPath, 'psci'), '0777');
+function use (version) {
+  var srcPath = path.join(paths.PSVM_VERSIONS, version, 'purescript'),
+      destPath = path.join(paths.PSVM_CURRENT_BIN),
+      mg = new glob("**/psc*", {cwd: srcPath}, function (er, files) {
+        files = files.map(function(match) {
+            util.copy(path.join(srcPath, match), path.join(destPath, match));
+            fs.chmodSync(path.join(destPath, match), '0777');
```

Comment:
Thanks :D
Last time I used Javascript extensively is in 2012.
I know the language, but a lot of stuff I've never used before (promises, etc).

I imagine I just copied that instance of `glob` out of some docs. If I remove that it gives me many fun errors.
I'm used to knowing my errors at compile time now, it's pretty cringeworthy to go back to Javascript's breaking at runtime and lack of info in the errors ^^;;
Feel free to fix this code as you want, I don't plan on learning node anyway, so this is as far as I can do without having to put in some learning time.